src/utils/fetch-config.ts

```diff
@@ -43,6 +43,10 @@ export async function fetchConfig (): Promise<void> {
   sessionStorage.setItem('CREATE_URL', createUrl)
   console.log('Set Create URL to: ' + createUrl)
 
+  const correctUrl = response.data['CORRECT_URL']
+  sessionStorage.setItem('CORRECT_URL', correctUrl)
+  console.log('Set Correct URL to: ' + correctUrl)
```

Comment:
Should this be console.info? I see both console.log and console.info statements in this file.

Reply:
Good catch. I think these should all be console.info. Thanks, fixed in next commit.